stl/inc/__msvc_int128.hpp

```diff
@@ -721,6 +718,39 @@ struct _Unsigned128 : _Base128 {
     using _Base128::_Base128;
     constexpr explicit _Unsigned128(const _Base128& _That) noexcept : _Base128{_That} {}
 
+#ifdef __clang__ // TRANSITION, Clang 16 or 17
+    constexpr explicit _Unsigned128(const float _Val) noexcept {
+        _Word[0] = static_cast<uint64_t>(_Val);
+        _Word[1] = static_cast<uint64_t>(_Val / 18446744073709551616.0f);
+    }
+
+    constexpr explicit _Unsigned128(const double _Val) noexcept {
+        _Word[0] = static_cast<uint64_t>(_Val);
+        _Word[1] = static_cast<uint64_t>(_Val / 18446744073709551616.0);
+    }
+
+    constexpr explicit _Unsigned128(const long double _Val) noexcept {
+        _Word[0] = static_cast<uint64_t>(_Val);
+        _Word[1] = static_cast<uint64_t>(_Val / 18446744073709551616.0L);
+    }

```

Comment:
`long double` is treated in the same way as `double` among the MSVC STL. Is it make sense to make such change _only_ here?

Reply:
You're right, but I'm slightly OCD and it kills me to see that only the `long double` version is different from the other two.....
Anyway, fixed at https://github.com/microsoft/STL/commit/e876bbd001b3c87765a83549678df63e788e2cfa